flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -148,6 +148,16 @@
 	public static final String TASK_MANAGER_NETWORK_DEFAULT_IO_MODE = "taskmanager.network.defaultIOMode";
 
 	/**
+	 * Initial backoff (in ms) for partition requests, which return partition not found.
+	 */
+	public static final String TASK_MANAGER_NETWORK_REQUEST_BACKOFF_INITIAL_MS = "taskmanager.network.requestBackoffInitial";
```

Comment:
For unit tests, you could try to pass them directly to the network environment.

For system integration tests, it is tricky. You may use some reflection magic modify the constants (sneaky, but test only).